examples/blog-starter/now.json

```diff
@@ -0,0 +1,11 @@
+{
+  "version": 2,
+  "name": "nextjs-blog-starter",
+  "alias": ["nextjs-blog-starter.now.sh"],
+  "builds": [{ "src": "package.json", "use": "@now/next" }],
+  "routes": [
+    { "src": "/feed.json", "dest": "/_next/static/feed.json" },
+    { "src": "/blog/(?<page>[^/]+)$", "dest": "/blog?page=$page" },
+    { "src": "/(.*)", "dest": "/$1" }
```

Comment:
Why is this route present?

Reply:
I think I pulled it from another project `now.json` file. I assume it is redundant to have it here, if that's the case I'll remove it.